### PR TITLE
premium page

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Extensive redesign of the premium page.

## WHY
To make it clearer why schools and districts should purchase premium instead of having individual teachers do it.

## HOW
React and CSS!

### Screenshots
Screen Shot 2021-04-01 at 3 31 12 PM
Screen Shot 2021-04-01 at 3 31 20 PM
Screen Shot 2021-04-01 at 3 31 34 PM
Screen Shot 2021-04-01 at 3 31 39 PM
Screen Shot 2021-04-01 at 3 31 44 PM
Screen Shot 2021-04-01 at 3 31 50 PM
Screen Shot 2021-04-01 at 3 31 53 PM
Screen Shot 2021-04-01 at 3 31 56 PM
Screen Shot 2021-04-01 at 3 31 59 PM
Screen Shot 2021-04-01 at 3 32 17 PM
Screen Shot 2021-04-01 at 3 32 24 PM
Screen Shot 2021-04-01 at 3 32 31 PM
Screen Shot 2021-04-01 at 3 32 32 PM
Screen Shot 2021-04-01 at 3 32 36 PM
Screen Shot 2021-04-01 at 3 32 43 PM
Screen Shot 2021-04-01 at 3 32 46 PM

### Notion Card Links
https://www.notion.so/quill/Update-the-premium-page-to-make-the-schools-and-district-offerings-more-compelling-7c2e539687c2441bbac2b7372e74f56f

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  YES
Have you deployed to Staging? | YES
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Design Review: If applicable, have you compared the coded design to the mockups? | YES

### CODE REVIEW

**Review by cissyxyu — APPROVED** [review]
LGTM!

File: services/QuillLMS/client/app/bundles/Teacher/components/premium/infoTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+
+import { Tooltip, } from '../../../Shared/index'
+
+const helpIconSrc = `${process.env.CDN_URL}/images/icons/icons-help.svg`
```
Comment (eadams17) [inline services/QuillLMS/client/app/bundles/Teacher/components/premium/infoTooltip.tsx:5]: I'm wondering if we should maybe just pull all these icon srcs we have throughout the codebase into the shared library at some point... what are your thoughts?

File: services/QuillLMS/client/app/bundles/Teacher/components/premium/premium_features_table.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+)
+
+const premiumFeaturesComponents = ({ independentPracticeActivityCount, lessonsActivityCount, diagnosticActivityCount, }) => premiumFeatures({ independentPracticeActivityCount, lessonsActivityCount, diagnosticActivityCount, }).map(section => (
+  <React.Fragment key={section.header}>
+    <div className="header-row">
+      <h4>{section.header}</h4>
+    </div>
+    {section.features.map(feature => <Row key={feature.label} {...feature} />)}
+  </React.Fragment>
+))
```
Comment (eadams17) [inline services/QuillLMS/client/app/bundles/Teacher/components/premium/premium_features_table.tsx:33]: love the way you handled this! I would just maybe break the `premiumFeatures` return out onto the next line so it's a bit easier to read

**Review by eadams17 — APPROVED** [review]
looks really good! so clean-- just left a few small comments

File: services/QuillLMS/client/app/bundles/Teacher/components/premium/infoTooltip.tsx (hunk 1 above)
Comment (emilia-friedberg) [reply services/QuillLMS/client/app/bundles/Teacher/components/premium/infoTooltip.tsx:5]: I think that's a really good idea. I'll start that in the teacher dashboard project.

**Pull request merged**